media-tv/xmltv/metadata.xml

```diff
@@ -1,29 +1,39 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
 <pkgmetadata>
-	<!-- maintainer-needed -->
+	<maintainer type="person">
+		<email>thebitpit@earthlink.net</email>
+		<name>Wilson Michaels</name>
+	</maintainer>
+	<maintainer type="project">
+		<email>proxy-maint@gentoo.org</email>
+		<name>Proxy Maintainers</name>
+	</maintainer>
 	<use>
 		<flag name="ar">Argentina tv listing grabber</flag>
 		<flag name="ch-search">Grabber for Switzerland</flag>
 		<flag name="dk">Denmark tv listing grabber</flag>
 		<flag name="dtvla">Latin America digital tv listing grabber</flag>
-		<flag name="es-laguiatv">Spain alternative grabber</flag>
+		<flag name="es-laguiatv">Spain alternative grabber [REMOVED 0.6.1]</flag>
```

Comment:
It's pretty obvious it's been removed when a new version doesn't have it anymore. Further more, this whole description needs to be removed when you remove the older version, so this doesn't make much sense.

Also see my suggestion to "complete" rehaul of these USe flags.

Reply:
Done